.mailmap

```diff
@@ -90,9 +90,9 @@ Ulrik Stervbo <ulriks@users.sourceforge.net>
 Stefano Gariazzo <steog88@gmail.com>
 Lee Patton <lpatton@users.sourceforge.net>
 Jörg Lenhard <joerg.lenhard@kau.se>
-<joerg.lenhard@kau.se> <joerg@lenhard.info>
```

Comment:
Please do not change this to the real name.

The intention is to map all different email addresses of one user to a single email address. The reason for that is that tools might have name + email or even the email only as unique key for code analysis. When having different emails, users might be treated as separate users.

Reply:
okay, the problem is just, that now if I run the `generate-authors` script a new name `Joerg Lenhard` appears. How can I do that properly?